src/test/scala/org/hammerlab/guacamole/likelihood/LikelihoodSuite.scala

```diff
@@ -18,32 +18,74 @@
 
 package org.hammerlab.guacamole.likelihood
 
-import org.hammerlab.guacamole.likelihood.LikelihoodUtil._
+import org.bdgenomics.adam.util.PhredUtils
 import org.hammerlab.guacamole.pileup.Pileup
 import org.hammerlab.guacamole.reads.MappedRead
-import org.hammerlab.guacamole.util.{GuacFunSuite, TestUtil}
-import org.hammerlab.guacamole.variants.Genotype
+import org.hammerlab.guacamole.util.{Bases, GuacFunSuite, TestUtil}
+import org.hammerlab.guacamole.variants.{Allele, Genotype}
 import org.scalatest.prop.TableDrivenPropertyChecks
 
 class LikelihoodSuite extends GuacFunSuite with TableDrivenPropertyChecks {
 
+  implicit lazy val reference = TestUtil.makeReference(sc, Seq(("chr1", 1, "C")))
```

Comment:
ah, it is unused here, good catch!

it gets used by #533 but just snuck in here; anyway I'll document it over there as well